pkg/machines/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -130,15 +131,17 @@ const StoragePoolTargetRow = ({ onValueChanged, dialogValues }) => {
                         <p className="text-danger">{_("Target path should not be empty")}</p>
                     </HelpBlock> }
                 </FormGroup>
+                <hr />
```

Comment:
Is that deliberate? That looks like an unrelated change.

Reply:
It is actually delibarate. Before the hr elemenmt was outside of the TargetRow component, and in case the component is not included (which is the case for iscsi-direct) we would have duplicate <hr>. I moved it inside as all other components.